py_ocsf_models/objects/account.py

```diff
@@ -36,4 +36,4 @@ class Account(BaseModel):
     type: Optional[str]
     type_id: TypeID
     uid: str
-    labels: Optional[list[str]]
+    labels: Optional[dict]
```

Comment:
This will not comply with the OCSF specification https://schema.ocsf.io/1.2.0/objects/account?extensions=. `labels` must be a String Array.